src/main/java/com/api/util/ApiSecurity/ApiSigning.java

```diff
@@ -353,7 +362,50 @@ public static PublicKey getPublicKeyFromX509Certificate(String publicCertificate
         log.debug("Exit :: getPublicKeyFromX509Certificate");
         return pk;
     }
-
+    
+    /**
+     * @param key
+     * @return
+     * @throws IOException
```

Comment:
The javadoc comment lack a description what this method do?

Reply:
updated javadoc